README.md

```diff
@@ -1,6 +1,142 @@
-# knora-ui Test Application
+# DSP-UI library (knora-ui)
 
-## Running Application in Productive Mode
+
+This is the demo and developing environment for DSP-UI library composed of 4 modules.
+
+The modules helps to create a graphical user interface, a web application to use [DSP-API](https://www.knora.org) in a quick and simple way. The modules are written in typescript to use them with [Angular](https://angular.io) (version 8). We decided to style the components and directives with [Angular Material design](https://material.angular.io).
+
+DSP-UI library implements [DSP-API-JS-LIB](https://www.npmjs.com/package/@knora/api) to connect with the Knora web API. DSP-API is a software framework for storing, sharing, and working with primary sources and data in the humanities.
+
+DSP-API and the DSP-UI library is [free software](http://www.gnu.org/philosophy/free-sw.en.html), released under the [GNU Affero General Public](http://www.gnu.org/licenses/agpl-3.0.en.html).
+
+This version of DSP-UI requires [DSP-API version ^12.0.0](https://github.com/dasch-swiss/knora-api/releases/tag/v12.0.0).
+
+## Published modules
+
+### knora-ui/core
+
+The core module contains configuration files and all variables needed to connect the DSP API.
+[read more...](https://dasch-swiss.github.io/knora-ui/modules/core)
+
+* * *
+
+### knora-ui/search
+
+Search module allows to make simple searches or extended searches in DSP-API. In the extended search, resource class and its properties related to one specific ontology are selected to create your query.
+[read more...](https://dasch-swiss.github.io/knora-ui/modules/search)
+
+* * *
+
+### knora-ui/viewer
+
+The viewer module contains object components to show the resource class representations from DSP-API, the gui-elements for the property values and different kind of view frameworks.
+[read more...](https://dasch-swiss.github.io/knora-ui/modules/viewer)
+
+* * *
+
+### knora-ui/action
```

Comment:
Looks good to me, but I'm not sure with the titles. We will have only one module at the end with four parts core, search, viewer and action

Reply:
We can use an other term, no problem! What about "library folders"?

Reply:
I don't know. It's not the term "modules" that bothers me. Probably we will keep several modules in the lib? This is a general question and should be well defined or clarified in the main readme of the repo. And btw the modules are not published yet.